### PR TITLE
Feature/cron job update

### PR DESCRIPTION
**Pull request by aslishemesh**
cron updates:
* change file locations
* add cron job for nosetests and fox runner

### CODE REVIEW

File: fox_cron.sh [hunk 1]
```diff
@@ -2,5 +2,6 @@
 # A simple script
 
 echo -e “Running nosetests…”
-nosetests
+./venv/bin/nosetests -s >> asli.log
```
Comment (caduri) [inline fox_cron.sh]: The cron doesn't need to run the tests

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# clone repo
```
Comment (caduri) [inline install.sh]: remove this file from repo

File: .gitignore [hunk 3]
```diff
@@ -3,4 +3,4 @@
 .DS_Store
 venv
 tests/.DS_Store
-
+asli.log
```
Comment (caduri) [inline .gitignore]: change this to `*.log`

**Review by caduri — APPROVED** [review]
LGTM

**Review by aslishemesh — COMMENTED** [review]
done

**Pull request closed without merging**